src/base/bittorrent/torrenthandle.cpp

```diff
@@ -475,20 +475,23 @@ void TorrentHandle::replaceTrackers(const QVector<TrackerEntry> &trackers)
     }
 }
 
-QList<QUrl> TorrentHandle::urlSeeds() const
+QVector<QUrl> TorrentHandle::urlSeeds() const
 {
-    QList<QUrl> urlSeeds;
     const std::set<std::string> seeds = m_nativeHandle.url_seeds();
 
+    QVector<QUrl> urlSeeds;
+    urlSeeds.reserve(seeds.size());
```

Comment:
Just noticed... You can do:
```c++
QVector<QUrl> urlSeeds {seeds.size()};
```

Reply:
yes, but that would become another usage pattern as it is now allocating `seeds.size()` elements  and later we would need to reassign them to our intended values, such as `urlSeeds[i] = ""` instead of `append()`. I would like to keep the current pattern.

Reply:
Ok.